pkg/config/env_var_test.go

```diff
@@ -205,15 +206,29 @@ func TestRemoveEnvVarsFromList(t *testing.T) {
 			expected: EnvVarList{},
 			keys:     []string{"foo"},
 		},
+		{
+			envVarList: EnvVarList{},
+			expected:   EnvVarList{},
+			keys:       []string{"nosuchenv"},
```

Comment:
In case of an error `nil` is expected @dharmit

Reply:
Or you can consider return an empty EnvVarList i.e. `newEnVarList` to retain the behavior